features/admin.tenants.v1/components/forms/add-tenant-wizard-form.tsx

```diff
@@ -153,6 +163,24 @@ export const AddTenantWizardForm: FunctionComponent<AddTenantWizardFormPropsInte
         }
     };
 
+    const updateDeploymentUnit = (deploymentUnitJson: string): void => {
```

Comment:
Let's add doc comment for this method. btw, for param `deploymentUnitJson`, are we passing a raw JSON string?

Reply:
We pass a json string parsed using JSON.stringify() method